set.mm

```diff
@@ -532760,6 +532528,73 @@ dependency is expressed in our hypothesis (called implicit
       CDBJABCDKL $.
   $}
 
+  ${
+    $d t x $.  $d x ph $.
+    $( Reverting substitution yields the original expression.  (Contributed by
+       Wolf Lammen, 5-Aug-2023.) $)
+    wl-ich-lem1 $p |- ( [ t / x ] [ x / t ] ph <-> ph ) $=
```

Comment:
Why not use sbid2v? It does not require t and x to be distinct.

Reply:
Because I did not notice it, when looking for variants of sbco2